Makefile

```diff
@@ -1,73 +1,66 @@
-WORKDIR      := $(shell pwd)
-NATIVEOS	 := $(shell go version | awk -F '[ /]' '{print $$4}')
-NATIVEARCH	 := $(shell go version | awk -F '[ /]' '{print $$5}')
-INTEGRATION  := postgresql
-BINARY_NAME   = nri-$(INTEGRATION)
-GO_PKGS      := $(shell go list ./... | grep -v "/vendor/")
-GO_FILES     := ./src/
-GOTOOLS       = github.com/kardianos/govendor \
-		gopkg.in/alecthomas/gometalinter.v2 \
-		github.com/axw/gocov/gocov \
-		github.com/AlekSi/gocov-xml \
+export PATH := $(PATH):$(GOPATH)/bin
+
+INTEGRATION     := postgresql
+
+BINARY_NAME      = nri-$(INTEGRATION)
+INTEGRATIONS_DIR = /var/db/newrelic-infra/newrelic-integrations/
+CONFIG_DIR       = /etc/newrelic-infra/integrations.d
+GO_FILES        := ./src/
+GOFLAGS			 = -mod=mod
```

Comment:
I've seen that we use `-mod=readonly` in other integrations but not sure why and which should be used

Reply:
https://github.com/newrelic/nri-nginx/blob/master/Makefile#L8